flink-python/bin/pyflink-udf-runner.sh

```diff
@@ -23,9 +23,22 @@ if [[ "$FLINK_TESTING" = "1" ]]; then
     FLINK_SOURCE_ROOT_DIR=`cd $ACTUAL_FLINK_HOME/../../../../../; pwd`
     FLINK_PYTHON="${FLINK_SOURCE_ROOT_DIR}/flink-python"
     if [[ ! -f "${FLINK_PYTHON}/pyflink/fn_execution/boot.py" ]]; then
-      # use pyflink source code to override the pyflink.zip in PYTHONPATH
-      # to ensure loading latest code
-      export PYTHONPATH="$FLINK_PYTHON:$PYTHONPATH"
+        # use pyflink source code to override the pyflink.zip in PYTHONPATH
+        # to ensure loading latest code
+        export PYTHONPATH="$FLINK_PYTHON:$PYTHONPATH"
+    fi
+fi
+
+if [[ "$python" = "" ]]; then
+    python="python"
+fi
+
+if [[ "$_PYTHON_WORKING_DIR" != "" ]]; then
```

Comment:
If we change it as above, sometimes users will not be able to access archives through relative paths because the python working directory is not set as expected.